backend/schemas.py

```
@@ -109,3 +111,68 @@ class UserUpdate(BaseModel):
+    brand_id: int
+    product_name: str
+    brand_name: str
+    order_size: str
+    order_quantity: int
+
+class OrderDetailOut(BaseModel):
+    order_id: int
+    status: str
+    created_at: datetime
+    bill_amount: Optional[Decimal]  # Using Decimal for currency
+    order_items: List[OrderItemDetail]
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)